models/cobalt/model.sdf

```diff
@@ -168,6 +173,75 @@
       </velocity_decay>
     </link>
 
+    <link name='hydrophones'>
+      <pose> 0 0 0 0 0 0 </pose>
+      <inertial>
+        <mass> 0.001 </mass>
+      </inertial>
+      <visual name='ref_hydrophone_visual'>
```

Comment:
These should probably not have a visual element to them since it's kinda ugly and it shows up on the camera.